src/AWS.Deploy.CLI/ConsoleUtilities.cs

```diff
@@ -307,6 +308,60 @@ public string AskUserForValue(string message, string defaultValue, bool allowEmp
             return userValue;
         }
 
+        public List<string> AskUserForList(List<string> listValues)
+        {
+            listValues ??= new List<string>();
+
+            if (listValues.Count == 0)
+            {
+                AskToAddListItem(listValues);
+                return listValues;
+            }
+
+            const string ADD = "Add new";
+            const string UPDATE = "Update existing";
+            const string DELETE = "Delete existing";
+            var operations = new List<string> { ADD, UPDATE, DELETE };
+
+            var selectedOperation = AskUserToChoose(operations, "Select which operation you want to perform:", ADD);
+
+            if (selectedOperation.Equals(ADD))
+                AskToAddListItem(listValues);
+
+            if (selectedOperation.Equals(UPDATE))
+                AskToUpdateListItem(listValues);
+
+            if (selectedOperation.Equals(DELETE))
+                AskToDeleteListItem(listValues);

```

Comment:
```suggestion
            if (selectedOperation.Equals(ADD))
                AskToAddListItem(listValues);
            else if (selectedOperation.Equals(UPDATE))
                AskToUpdateListItem(listValues);
            else if (selectedOperation.Equals(DELETE))
                AskToDeleteListItem(listValues);
```

slightly better, there is no worth doing all the checks after first successful check.